docker_client_test.go

```diff
@@ -32,8 +33,8 @@ func TestGetDockerInfo(t *testing.T) {
 			go func() {
 				defer wg.Done()
 				info, err := c.Info(ctx)
-				require.NoError(t, err)
-				require.NotNil(t, info)
+				assert.NoError(t, err)
+				assert.NotNil(t, info)
```

Comment:
suggestion: keep the original as we want to standardise on require, this is a good example why: if there is an error info will be will, so you'd get two errors instead of just the one.

That said this is broken as its in a go routine, but we can fix that seperately.